### DISCUSSION TITLE
Not possible to open data-hub configuration from PimcoreCIHubAdapterBundle

### DISCUSSION THREAD

**Discussion by dpfahlbusch** [BrandOriented/PimcoreCiHubConnector#68] [Q&A]
Currently it seems not possible to open a data-hub configuration from the PimcoreCiHubAdapterBundle.
Everytime I try to open the configuration in the backend I get the following error:

![image](https://github.com/user-attachments/assets/886cffaa-0304-40d8-bd1f-f13882b15c0b)

Both bundles are setup correct.

We use Pimcore 11.

**Comment by dpfahlbusch** [comment]
Seems like the error comes from this file: 'src/PimcoreCIHubAdapterBundle/Resources/public/pimcore/js/config-item.js'
From this line of code:
pimcore.plugin.pimcoreCIHubAdapterBundle.configuration.configItem = Class.create(pimcore.plugin.simpleRestAdapterBundle.configuration.configItem, {


simpleRestAdapterBundle seems not to exist as plugin or the name has changed

**Comment by johanlindgren** [comment]
I had the same problem and got it working after having the below bundles installed and ordered the way below 

PimcoreApplicationLoggerBundle
PimcoreDataHubBundle
PimcoreDataHubSimpleRestBundle
SimpleRESTAdapterBundle
PimcoreCIHubAdapterBundle
PimcoreElasticsearchClientBundle

I belive its important to have SimpleRESTAdapterBundle before PimcoreCIHubAdapterBundle in the list of bundles
